Makefile

```diff
@@ -67,9 +67,9 @@ FUZZER_BUILD ?= 0
 # NOTE: when changing any value below, you'll need to adjust WORKSPACE file by hand:
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
-OV_SOURCE_BRANCH ?= 4d86daa2638eeecee57d62db04c18d6c59e5eb07  # 19 June main
-OV_CONTRIB_BRANCH ?= e6eb43a32c98a04162a921a80d89f82b30910973  # 19 June main
-OV_TOKENIZERS_BRANCH ?= 0dcc805651b7744cf8340c4654cc3ea7523b28b8  # 19 June main
+OV_SOURCE_BRANCH ?= b80e30c4d7901175e1155f8d156252d350f00558  # 27 June main
+OV_CONTRIB_BRANCH ?= e6eb43a32c98a04162a921a80d89f82b30910973  # 13 June main
```

Comment:
Why date in the comment changes but the commit sha doesn't?

Reply:
before it was the date of update of that param, now is the date of the commit